embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/worker/NetworkRequestRunnable.kt

```diff
@@ -0,0 +1,40 @@
+package io.embrace.android.embracesdk.worker
+
+import io.embrace.android.embracesdk.comms.api.ApiRequest
+import java.util.concurrent.PriorityBlockingQueue
+import java.util.concurrent.atomic.AtomicLong
+
+private val counter = AtomicLong(0)
```

Comment:
Any reason this isn't an `AtomicInteger` if you're just converting it to an Int anyway?

Reply:
Force of habit - I forgot that `Comparator` returns an `Int`. Switched to using `AtomicInteger` instead